pkg/apis/istio/v1beta1/istio_types.go

```diff
@@ -616,6 +616,16 @@ type IstioSpec struct {
 	// Whether to restrict the applications namespace the controller manages
 	WatchOneNamespace bool `json:"watchOneNamespace,omitempty"`
 
+	// Under Kubernetes v1.17.0 it is not allowed to use the system-cluster-critical and system-node-critical
```

Comment:
```suggestion
	// Prior to Kubernetes v1.17.0 it was not allowed to use the system-cluster-critical and system-node-critical
```
Or maybe "before", "below ... it is not allowed". I think "Under" could be interpreted as "on"

Reply:
You are right, thanks!